packages/apidom-ls/src/config/openapi/components/lint/example--deprecated.ts

```diff
@@ -0,0 +1,34 @@
+import { DiagnosticSeverity } from 'vscode-languageserver-types';
```

Comment:
This should be moved to [src/config/common/schema/lint](https://github.com/swagger-api/apidom/tree/aa317d408ce3687ff3dfa9bb1a14d7c6486891bf/packages/apidom-ls/src/config/common/schema/lint)

Reply:
I thought I did that already. My bad. Fixed.